pkgs/development/libraries/openssl/default.nix

```diff
@@ -44,10 +44,12 @@ let
     # TODO(@Ericson2314): Improve with mass rebuild
     configurePlatforms = [];
     configureScript = {
-        x86_64-darwin  = "./Configure darwin64-x86_64-cc";
-        x86_64-solaris = "./Configure solaris64-x86_64-gcc";
         armv6l-linux = "./Configure linux-armv4 -march=armv6";
         armv7l-linux = "./Configure linux-armv4 -march=armv7-a";
+        x86-linux = "./Configure linux-x86";
```

Comment:
Is `"x86-linux"` a valid value for `stdenv.hostPlatform.system`?

Reply:
I am so damned. It should be `i686-linux`, sorry.